### PR TITLE
Update IC commit

### PR DESCRIPTION
**Pull request by sa-github-api**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

Changes from Action "Update IC"

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd please review

Automatic changes, the readme was to trigger the CI, it got stuck.

**Comment by dskloetd** [comment]
FYI, you can do `git commit -m empty --allow-empty` to trigger CI without changes.

Are you going to under the readme change?

**Comment by lmuntaner** [comment]
> FYI, you can do `git commit -m empty --allow-empty` to trigger CI without changes.
> 
> Are you going to undo the readme change?

Ah, thanks!

Reverted!

File: README.md
Comment (dskloetd) [inline README.md:1]: It doesn't look reverted to me?

**Review by dskloetd — APPROVED** [review]

File: README.md
Comment (lmuntaner) [reply README.md:1]: done, thanks!

**Comment by lmuntaner** [comment]
Already updated in #236

**Pull request closed without merging**